capactio/capact
# Secure gateway with BasicAuth

**Description**

Changes proposed in this pull request:
- enable BasicAuth on POST /graphql gateway endpoint
- add configuration options for BasicAuth
- *additional* - don't duplicate local voltron entries in `/etc/hosts`

**Test changes**
1. Start environment with `make dev-cluster`
2. Go to `https://gateway.voltron.local/`
3. Run a query with no headers, you should get an error
4. Add headers:
```
{
  "Authorization": "Basic Z3JhcGhxbDp0MHBfczNjcjN0"
}
```
5. Run again a query. You should get a proper response

**Related issue(s)**
